app/services/calculate_reaction_points.rb

```diff
@@ -0,0 +1,76 @@
+class CalculateReactionPoints
```

Comment:
I would also appreciate a comment on top of the file with a brief description where we calculate reaction points (as I see now, before saving the reaction).

Reply:
I see your point. Personally, I'm comfortable with using plural because I see it as a service "related to reactions logic", not necessarily related to multiple reactions. It's not an ideal solution but I would be uncomfortable if all our services would be in the root of the `services` folder :D I'm also comfortable with the "verbs" convention (because maybe it was me who had introduced it :D, informally)
For this particular pr I wouldn't require changes if you prefer the current naming.